librz/core/cmd_descs/cmd_descs.yaml

```diff
@@ -199,6 +199,11 @@ commands:
         type: RZ_CMD_ARG_TYPE_CHOICES
         optional: true
         choices: ["32", "64"]
+    details:
+      - name: "Settings of this command"
+        entries:
+          - text: "Check out the settings for this command with 'el basefind.'"
+            comment: ""
```

Comment:
@wargio You and me tend to put these messages, which only uses one of them, into `text` or `comment`. I prefer `text`, because it has a brighter color with the default theme for me. But with the search I remember you preferred `comment`. We should probably agree on one.

Reply:
As you prefer.